docs/specs/multi-registry/multi_registry.md

```diff
@@ -0,0 +1,366 @@
+  registry more readable and maintainable. Weaver will report an error if different aliases are used for the same
+  registry in the scope of a single registry.
+
+#### Do we allow an alias defined in one semconv file to be visible in another semconv file?
+- Short answer: Yes, under certain conditions.
+- Long answer: An alias defined in a file containing `global_imports` section is visible in all the files of the
+  registry. Such an alias is called a global alias. A global alias is useful when the same imported registry is
+  used in multiple files within the same registry. In this situation, defining a global alias simplifies the
+  management of dependencies, as updating to a new registry version will only require a single change in the
+  registry file containing the global aliases. Weaver will report an error if an alias not defined locally or as
+  a global alias is used in a semconv file.
+
+#### Do we allow importing the same registry with different versions?
+- Short answer: No for consistency.
+- Long answer: Although technically it is possible to support multiple versions of an imported registry, it is a
+  good practice to align the versions inside the same registry. This practice make the overall registry more
+  consistent and easy to reason about. Weaver will report an error if different versions are used for the same
+  registry in the scope of a single registry.
+
+#### Do we allow reference of experimental entities across registries?
+- Short answer: No for backward compatibility.
+- Long answer: Experimental entities are not meant to be used in production. Experimental entities are subject to
+  change or removal without any notice in the next version of the registry. The type of an experimental entity can
+  be changed. Weaver will detect and report an error if an experimental entity is referenced across registries.
+
+#### Do we allow circular references between registries?
+- Short answer: No.
+- Long answer: Circular references are already prohibited inside a registry, so circular references between
+  registries can lead to the kind of problems (e.g. unresolvable references). Weaver will detect and report
+  an error if a circular reference is detected between registries.
+
+#### Do we allow references to groups or attributes defined in an imported registry?
+- Short answer: Yes.
+- Long answer: References to groups or attributes defined in an imported registry are allowed. This feature
+  allows a registry to extend or override the attributes of an imported group. Weaver will detect and report an
+  error if a reference is made to an entity that is not defined in the imported registry.
+
+### Open Questions:
+- Can we override any field of a group defined in an imported registry? No for `type` field, what about the other
+  fields?
+- Is there a relationship to define between the instrumentation scope name and version and the registry?
```

Comment:
instrumentation scope is a random string - name of the library/class/component that produced the telemetry, I can change which component/class emits my logs/events, but this should have no effect on the consumer of the telemetry.

I think we might change it at some point, but that's the current state of affairs.